guide/content/Brand.md

```diff
@@ -46,7 +46,7 @@ Depending on the length of the sub-brand, the font size might differ slightly.
 We love it when people show the OONI logo, however we kindly ask you to not use
 it in the following ways.
 
-![OONI Logo don'ts](./images/XXX.svg)
+![OONI Logo don'ts](./images/Donts.svg)
```

Comment:
You forgot to upload the image `Donts.svg` to the `images` folder.

Reply:
@hellais here we go https://github.com/OpenObservatory/design/pull/31/

Reply:
I meant you should add it to this branch. PR#31 also seems to include some changes that are unrelated. I guess I will merge #21 into #22 when I get back to proper internet.